+++

headless = true  # This file represents a page section.
active = true  # Activate this widget? true/false
weight = 80  # Order that this section will appear.

title = "Publications"
subtitle = ""

[design]
  # Toggle between the various page layout types.
  #   1 = List
  #   2 = Compact
  #   3 = Card
  #   4 = Citation (publication only)


[design.background]
  # Apply a background color, gradient, or image.
  #   Uncomment (by removing `#`) an option to apply it.
  #   Choose a light or dark text color by setting `text_color_light`.
  #   Any HTML color name or Hex value is valid.
  
  # Background color.
  # color = "navy"
  
  # Background gradient.
  # gradient_start = "DeepSkyBlue"
  # gradient_end = "SkyBlue"
  
  # Background image.
  # image = "background.jpg"  # Name of image in `static/img/`.
  # image_darken = 0.6  # Darken the image? Range 0-1 where 0 is transparent and 1 is opaque.

  # Text color (true=light or false=dark).
  # text_color_light = true  
  
[advanced]
 # Custom CSS. 
 css_style = ""
 
 # CSS class.
 css_class = ""
+++

1. Kim R S, **Levy Karin E**, Mirdita M, Chikhi R & Steinegger M. BFVD—a large repository of predicted viral protein structures. Nucleic Acids Research. 2024; gkae1119. [PDF](https://doi.org/10.1093/nar/gkae1119).
+. Kim W, Mirdita M, **Levy Karin E**, Gilchrist CL, Schweke H, Söding J, Levy E & Steinegger M. Rapid and Sensitive Protein Complex Alignment with Foldseek-Multimer. BioRxiv. 2024. [PDF](https://www.biorxiv.org/content/10.1101/2024.04.14.589414v4)
- Kim G^, Lee S^, **Levy Karin E**^, Kim H, Moriwaki Y, Ovchinnikov S, Steinegger M & Mirdita M. Easy and accurate protein structure prediction using ColabFold. Nature Protocols. 2024; s41596-024-01060-5. [PDF](https://rdcu.be/dWViQ)
- Lee S, Kim G, **Levy Karin E**, Mirdita M, Park S, Chikhi R, Babaian A, Kryshtafovych A, & Steinegger M. Petabase-Scale Homology Search for Structure Prediction. Cold Spring Harbor Perspectives in Biology. 2024; a041465. [PDF](http://m.cshperspectives.cshlp.org/content/early/2024/02/05/cshperspect.a041465.full.pdf)
- Raghavan V, Eichele G, Larink O, **Levy Karin E**, & Söding J. RNA sequencing indicates widespread conservation of circadian clocks in marine zooplankton. NAR Genomics and Bioinformatics. 2023. lqad007. [PDF](https://academic.oup.com/nargab/article-pdf/5/1/lqad007/48992000/lqad007.pdf)
- Mirdita M, Steinegger M, Breitwieser F, Soeding J^, & **Levy Karin E**^. Fast and sensitive taxonomic assignment to metagenomic contigs. Bioinformatics. 2021; btab184. [PDF](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab184/6178277)
- Zhang R, Mirdita M, **Levy Karin E**, Norroy C, Galiez C, & Soeding J. SpacePHARER: Sensitive identification of phages from CRISPR spacers in prokaryotic hosts. Bioinformatics. 2021; btab222. [PDF](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab222/6207963)
- Halabi K, **Levy Karin E**^, Guéguen L, & Mayrose I^. TraitRELAX - A codon model for associating phenotypic traits with altered selective patterns of sequence evolution. Systematic Biology. 2020; syaa087. [PDF](https://academic.oup.com/sysbio/advance-article-abstract/doi/10.1093/sysbio/syaa087/6012374?redirectedFrom=fulltext)
- **Levy Karin E**, Mirdita M, & Soeding J. MetaEuk – sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics. Microbiome. 2020; 8:48. [PDF](https://rdcu.be/b3ozK)
- **Levy Karin E**, Ashkenazy H, Hein J, & Pupko T. A simulation-based approach to statistical alignment. Systematic Biology. 2019; 2:252-266. [PDF](https://academic.oup.com/sysbio/article-pdf/68/2/252/27739173/syy059.pdf)
- Ashkenazy H, Sela I, **Levy Karin E**, Landan G, & Pupko T. Multiple sequence alignment averaging improves phylogeny reconstruction. Systematic Biology. 2019; 1:117–130. [PDF](https://academic.oup.com/sysbio/article-pdf/68/1/117/27127304/syy036.pdf)
- Lavi B, **Levy Karin E**, Pupko T, & Hazkani-Covo E. The prevalence and evolutionary conservation of inverted repeats in proteobacteria. Genome Biology and Evolution. 2018; 3:918–927. [PDF](https://academic.oup.com/gbe/article-pdf/10/3/918/24820113/evy044.pdf)
- Mushegian A, **Levy Karin E**, & Pupko T. Sequence analysis of malacoherpesvirus proteins: pan-herpesvirus capsid module and replication enzymes with an ancient connection to - "Megavirales". Virology. 2018; 513: 114-128. [PDF](https://www.sciencedirect.com/science/article/pii/S0042682217303513/pdfft?md5=368336fcd5bcf59771691469ceb7f34b&pid=1-s2.0-S0042682217303513-main.pdf)
- **Levy Karin E**^, Ashkenazy H^, Wicke S, Pupko T, & Mayrose I. TraitRateProp: a web server for the detection of associations between phenotypic trait changes and specific sequence sites. Nucleic Acids Research. 2017; 45:W260-W264. [PDF](https://academic.oup.com/nar/article-pdf/45/W1/W260/18137235/gkx288.pdf)
- Ashkenazy H^, **Levy Karin E**^, Mertens Z, Cartwright R, & Pupko T. SpartaABC: a web server to simulate sequences with indel parameters inferred using an approximate Bayesian computation algorithm. Nucleic Acids Research. 2017; 45:W453-W457. [PDF](https://academic.oup.com/nar/article-pdf/45/W1/W453/18137313/gkx322.pdf)
- **Levy Karin E**, Wicke S, Pupko T, & Mayrose I. An integrated model of phenotypic trait changes and site-specific sequence evolution. Systematic Biology. 2017; 6:917–933. [PDF](https://academic.oup.com/sysbio/article-pdf/66/6/917/21192508/syx032.pdf)
- **Levy Karin E**^, Shkedy D^, Ashkenazy H, Cartwright R, & Pupko T. Inferring rates and length-distributions of indels using approximate Bayesian computation. Genome Biology and Evolution. 2017; 9:1280-1294. [PDF](https://academic.oup.com/gbe/article-pdf/9/5/1280/19270382/evx084.pdf)
- Preisner H, **Levy Karin E**, Poschmann G, Stühler K, Pupko T, & Gould S. The cytoskeleton of parabasalian parasites comprises proteins that share properties common to intermediate filament proteins. Protist. 2016; 167:526–543. [PDF](https://www.sciencedirect.com/science/article/pii/S1434461016300517/pdfft?md5=ea5865a405b4155f9dc2c8377887b631&pid=1-s2.0-S1434461016300517-main.pdf)
- **Levy Karin E**^, Rabin A^, Ashkenazy H, Shkedy D, Avram O, Cartwright R, & Pupko T. Inferring indel parameters using a simulation-based approach. Genome Biology and Evolution. 2015; 7:3226-3238. [PDF](https://academic.oup.com/gbe/article-pdf/7/12/3226/17924188/evv212.pdf)
- **Levy Karin E**, Susko E, & Pupko T. Alignment errors strongly impact likelihood-based tests for comparing topologies. Molecular Biology and Evolution. 2014; 31:3057-3067. [PDF](https://academic.oup.com/mbe/article-pdf/31/11/3057/13167922/msu231.pdf)
- Cohen O, Ashkenazy H, **Levy Karin E**, Burstein D, & Pupko T. CoPAP: co-evolution of presence-absence patterns. Nucleic Acids Research. 2013; 41:W232-W237. [PDF](https://academic.oup.com/nar/article-pdf/41/W1/W232/3872430/gkt471.pdf)
  
